python/ray/serve/_private/router.py

```diff
@@ -45,6 +49,9 @@
 
 logger = logging.getLogger(SERVE_LOGGER_NAME)
 
+# Used to only print a single warning when users pass starlette requests via handle.
```

Comment:
```suggestion
# Used only to print a single warning when users pass starlette requests via handle.
```

Reply:
The phrasing is as-is to specify that it's used "used to only print a single" versus printing multiple.

Not that this variable is "only used to print" a warning

Reply:
Oh got it, could we phrase it like this:
```suggestion
# Used to print only a single warning when users pass starlette requests via handle.
```

Reply:
yeah not going to wait for another build for it though, if it fails i'll update it

Reply:
That sounds good.